helm/prometheus-rules/templates/alerting-rules/calico.rules.yml

```diff
@@ -35,3 +35,35 @@ spec:
         severity: notify
         team: ludacris
         topic: kubernetes
+{{- if eq .Values.managementCluster.provider.kind "kvm" }}
+    - alert: CalicoNodeFailingToSaveIptables
+      annotations:
+        description: '{{`calico-node {{ $labels.namespace}}/{{ $labels.pod_name }} on {{ $labels.cluster_id}}/{{ $labels.cluster }} is failing to save iptables rules.`}}'
+        opsrecipe: calico-iptables-failing/
+      expr: increase(felix_iptables_save_errors[10m]) > 0
+      for: 10m
+      labels:
+        area: kaas
+        cancel_if_cluster_status_creating: "true"
+        cancel_if_cluster_status_deleting: "true"
+        cancel_if_cluster_status_updating: "true"
+        cancel_if_outside_working_hours: "true"
+        severity: notify
```

Comment:
Just notify for now?

Reply:
yeah, i'm unsure as to how much noise it will create - my plan was to see how it goes and then bump the severity once we're happy